src/OpenLoco/src/Ui/Windows/BuildVehicle.cpp

```diff
@@ -326,19 +330,16 @@ namespace OpenLoco::Ui::Windows::BuildVehicle
         return window;
     }
 
-    /* 0x4C1AF7
-     * depending on flags (1<<31) vehicle is a tab id or a VehicleHead id
-     */
-    Window* open(uint32_t vehicle, uint32_t flags)
+    // 0x004C1AF7
+    Window* open(uint32_t vehicleId, bool isTabId)
```

Comment:
This is only a vehicleId if isTabId is false. I'd prefer we split the function up into a tab and vehicleId version but failing that we should use a parameter name that is clear it could be either.